DotNET/Complex Flow Examples/ocr-with-extract-text.cs

```diff
@@ -0,0 +1,62 @@
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+class Program
```

Comment:
The class name does not need to match the file name (unlike Java), but we could call this something more interesting (`OcrWithExtractText`?)